stable/weave-scope/values.yaml

```diff
@@ -123,3 +131,7 @@ weave-scope-cluster-agent:
       # cpu: ""
       # weave-scope-cluster-agent.resources.limits.memory: memory limit in MiB (Mi)
       # memory: ""
+  # Security contexts options for weave-scope-cluster-agent. Both to true by default.
+  securityContext:
+    runAsNonRoot: true
+    readOnlyRootFilesystem: true
```

Comment:
There should be a new line at the end of the file. Can you please add that. This is a cause for a CI lint failure.